playbulb-candle/sw.js

```diff
@@ -1,4 +1,4 @@
-const version = "9";
+const version = "10";
```

Comment:
Using the stale while revalidate pattern, you don't need to update this version I think. Every load will return from cache - but at the same time go to the network and obtain a fresh network version.